WasatchNET/FeatureIdentification.cs

```diff
@@ -3,7 +3,7 @@
 namespace WasatchNET
 {
     /// <summary>the fundamental electronic board configurations supported by our spectrometers</summary>
-    public enum BOARD_TYPES { RAMAN_FX2, INGAAS_FX2, DRAGSTER_FX3, ARM, ERROR };
+    public enum BOARD_TYPES { RAMAN_FX2, INGAAS_FX2, DRAGSTER_FX3, ARM, STROKER,ERROR };

```

Comment:
Please add a comment that STROKER units are FX2-based, pre-FID protocol, don't have PID 0x1000 and don't have physical EEPROMs.